election_reminder.py

```diff
@@ -1,34 +1,45 @@
+#! /usr/bin/env python
 # -*- coding: utf-8 -*-
+# vim: set noexpandtab ts=2:
```

Comment:
🤔

Reply:
My vim config and your choice of whitespace in python were not getting along.